modules/nw-throughput-troubleshoot.adoc

```diff
@@ -1,50 +1,39 @@
 // Module filename: nw-throughput-troubleshoot.adoc
 // Module included in the following assemblies:
-// * networking/configuring-routing.adoc
+// * networking/routes/route-configuration.adoc
 
+:_content-type: CONCEPT
 [id="nw-throughput-troubleshoot_{context}"]
-= Troubleshooting throughput issues
-Sometimes applications deployed through {product-title} can cause
-network throughput issues such as unusually high latency between
-specific services.
+= Throughput issue troubleshooting methods
 
-Use the following methods to analyze performance issues if pod logs do not
-reveal any cause of the problem:
+Sometimes applications deployed through {product-title} can cause network throughput issues such as unusually high latency between specific services.
 
-* Use a packet analyzer, such as ping or link:http://www.tcpdump.org/[tcpdump]
-to analyze traffic between a pod and its node.
+Use the following methods to analyze performance issues if pod logs do not reveal any cause of the problem:
+
+* Use a packet analyzer, such as ping or link:http://www.tcpdump.org/[tcpdump] to analyze traffic between a pod and its node.
```

Comment:
Few suggestions the statement:
- Capturing tcpdump inside a pod and the on the node has different step and the implementation can differ specifically with dealing with pod level capture, I believe should include the general method such as article https://access.redhat.com/solutions/4569211 for a reference so the the reader will have better understanding of it. 
- The same applies to capturing tcpdump from the nodes, we have a good article : https://access.redhat.com/solutions/5074041 for it to be referenced. 
- For iperf testing https://access.redhat.com/solutions/6129701 will be a better reference to be added

Reply:
excellent, thanks for the feedback Arvind

Reply:
@quarterpin i've implemented your feedback, could you PTAL again for ack? Thank you!